pkg/config/constants/filters.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package constants
```

Comment:
add this to xds package instead of a new constants package?

Reply:
it's not a new package
BTW, `WasmHTTPFilterType` and `TypedStructType` reference by `istio.io/istio/pkg/wasm`

Reply:
Sorry I did not understand. I am suggesting to move this file to /pkg/config/xds/filters.go under xds package